src/App.test.js

```diff
@@ -1,8 +1,11 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import { render } from '@testing-library/react';
+import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
 
+// import TestRenderer from 'react-test-renderer';
 test('renders learn react link', () => {
```

Comment:
[Optional] It's considered best practice to organize all your test files in a single `__tests__` folder in your `src` directory. This way, your test files are easy to locate and categorize. I strongly suggest you implement this practise👍